packages/tabs/src/Tabs.ts

```diff
@@ -124,6 +125,9 @@ export class Tabs extends SizedMixin(Focusable, { noDefaultSize: true }) {
     @property({ attribute: false })
     public shouldAnimate = false;
 
+    @property({ attribute: false })
+    public shouldAutoscroll = false;
```

Comment:
I feel like I asked this, but can't find it in previous comments, with this structure, does this need to be opt-in? Feels like we should always scroll to the item. Most time it wouldn't do anything, but the couple of times it did, it would be a win for everyone, right?